### PR TITLE
Version 1.0.4 of IBM Federated Learning library

### PR DESCRIPTION
**Pull request by shashank215r**
### Changes
* Added multiple bias mitigation methods for training fairer FL models.
* Bug fixes in Tensorflow 2.1 Model wrapper.
### New Functionality
* Support Bias Mitigation methods.
   - Local Reweighing 
   - Global Reweighing with Differential Privacy
   - Federated Prejudice Removal 
* New Algorithm: FedProx

### CODE REVIEW

**Review by Yi-Zoey — APPROVED** [review]
Thanks for the update!

File: README.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ For a particular ML model, you can select multiple types of fusion algorithms:
 |	Reinforcement Learning RLLib models	|	Iterative Average        |
 |                                     |	FedAvg [McMahan et al.](https://arxiv.org/pdf/1602.05629.pdf)  |
 |Linear classifiers with SGD | Iterative Average |
+|                            | Local Reweighing [Abay et al.](https://arxiv.org/abs/2012.02447)|
+|                            | Global Reweighing with Differetial Privacy [Abay et al.](https://arxiv.org/abs/2012.02447)|
+|Logistic Regression | Federated Prejudice Removal [Abay et al.](https://arxiv.org/abs/2012.02447) |
```
Comment (Nathalie-B) [inline README.md:46]: Why is this listed by model? They are applicable for all model types. Can you pls update? 

Also, we are missing: FedProx: [Tian Li et al.](https://arxiv.org/pdf/1812.06127.pdf).

**Review by Nathalie-B — CHANGES_REQUESTED** [review]
Hi @shashank215r Thanks for the PR. Just one change requested. Thanks

**Review by Nathalie-B — APPROVED** [review]
Thanks, I made a minor edit. I think we are ready to go 👍

**Pull request merged**